net/pfSense-pkg-freeradius2/files/usr/local/pkg/freeradius.inc

```diff
@@ -4517,6 +4531,15 @@ function freeradius_validate_eap($post, &$input_errors) {
 		$input_errors[] = "The 'Maximum Sessions Tracking Per Server' field must contain an integer value.";
 	}
 
+	// Certificates for TLS
+	if ($post['ssl_ca_cert'] == 'none') {
```

Comment:
If "None" is not valid why even offer it in the drop-down? Does that break anything?